style.css

```diff
@@ -87,6 +91,113 @@ p {
   width: 80%;
 }
 
+.modal {
+  width: 90%;
+  display: none;
+  flex-direction: column;
+  margin: 0.8rem auto;
+  position: fixed;
+  left: 0;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  background-color: #fff;
+  padding: 0.7rem;
+  border-radius: 1rem;
+  border: 1px solid #dfe1e6;
+}
```

Comment:
- [ ] **[OPTIONAL]** Great job incorporating all the changes requested by the previous reviewer! I have one more suggestion **that is entirely optional** but **could enhance the overall appearance and align it closely with the [provided template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?type=design&node-id=0-1&t=dplvWZodUXxA9pZz-0)**. You might consider **adding a blurry background when the pop-up is open on both desktop and mobile devices**. This small addition can provide a more polished and visually appealing experience. 🤓

  ![image](https://github.com/MohamedRoshdy2021/mobile-first-project/assets/107952724/3be98c8f-c4e7-4a3e-8679-249a3efc4a78)